conf/broker.conf

```diff
@@ -1211,6 +1211,9 @@ replicationConnectionsPerBroker=16
 # Replicator producer queue size
 replicationProducerQueueSize=1000
 
+# Replicator client memory usage limit (in bytes, disable with value 0)
+replicationClientMemoryLimitBytes=0
```

Comment:
@lhotari Another possibility would be to force it to 0, without configuration, until a better flow control is implemented

Reply:
I have taken this approach.